README.md

```diff
@@ -5,10 +5,18 @@
 ### gitbase & engine
 
 - [ ] Improve gitbase performance
+    - [ ] Improve gitbase indexes
```

Comment:
But This aren't only this the improvements we need to do for this Q

Reply:
Should we add here go-git performance improvements?

Reply:
This will need a proper broke up after a design meeting, so maybe doesn't make sense write the KR now